src/prompts/workItemInitPrompt.template.md

```diff
@@ -112,7 +112,16 @@ After creating the workflow structure and checking out the target branch, commit
 
 This ensures the workflow context is tracked in version control from the start.
 
-### 8. Open WorkflowContext.md
+### 8. Push Target Branch to Remote
+
+Push target branch: `git push -u origin {{TARGET_BRANCH}}` (ensures target exists for PRs; acceptable if already exists)
+
+**Error Handling**:
```

Comment:
No need to add error handling - the agent will determine what to do. Remove this section.